packages/mask/src/plugins/WalletService/services/wallet/index.ts

```diff
@@ -193,7 +193,7 @@ export async function exportMnemonicWords(address: string, unverifiedPassword?:
 
 export async function exportPrivateKey(address: string, unverifiedPassword?: string) {
     if (unverifiedPassword) await password.verifyPasswordRequired(unverifiedPassword)
-    const password_ = await password.INTERNAL_getPasswordRequired()
+    const password_ = (await password.INTERNAL_getPasswordRequired()) ?? unverifiedPassword
```

Comment:
The user won't know the mastery password. It cannot be treated as a password here.

Reply:
It does not be treated as a password here.